src/components/Datatable/Datatable.tsx

```diff
@@ -286,6 +304,7 @@ Datatable.propTypes = {
   }),
   batchActions: PropTypes.arrayOf(ActionPropType),
   onDataFetch: PropTypes.func,
+  onSuggestionsFetch: PropTypes.func,
```

Comment:
this property belongs more to `controlsConfig` prop not to the Datatable root props

Reply:
this makes sense. I moved it to controlsConfig on L35 Datatable.types because `searchConfig?: SearchBarProps;`

Reply:
you can extend SearchBarProps, like `SearchBarProp & { onSuggestionsFetch: ... }` or `interface xxx extends SearchBarProps { onSuggestionsFetch: ... }`. it belong to `searchConfig`